src/backend/vulkan/src/conv.rs

```diff
@@ -466,7 +466,25 @@ pub fn map_device_features(features: Features) -> vk::PhysicalDeviceFeatures {
         .sparse_residency_aliased(features.contains(Features::SPARSE_RESIDENCY_ALIASED))
         .variable_multisample_rate(features.contains(Features::VARIABLE_MULTISAMPLE_RATE))
         .inherited_queries(features.contains(Features::INHERITED_QUERIES))
-        .build()
+        .build();
+
+    let extensions = crate::DeviceCreationExtensions {
```

Comment:
Would it make sense to have this structure containing the `core` as well? Returning a tuple from this function doesn't really give us anything :)

Reply:
Definitely